build.sbt

```diff
@@ -0,0 +1,28 @@
+name := "akka-http-websocket-activator-template"
+organization :=  "Ordina"
```

Comment:
I think we should kill the whole template. This is never going to work :disappointed: